gflownet/envs/tree.py

```diff
@@ -254,22 +274,27 @@ def __init__(
         attributes_root[Attribute.TYPE] = NodeType.CLASSIFIER
         attributes_root[Attribute.FEATURE] = -1
         attributes_root[Attribute.THRESHOLD] = -1
-        attributes_root[Attribute.CLASS] = Counter(self.y_train).most_common()[0][0]
+        self.default_class = Counter(self.y_train).most_common()[0][0]
+        attributes_root[Attribute.CLASS] = self.default_class
         attributes_root[Attribute.ACTIVE] = Status.INACTIVE
 
         # End-of-sequence action.
-        self.eos = (-1, -1)
+        self.eos = (-1, -1, -1)
 
         # Conversions
-        # TODO: add functionality to select conversion type depending on config
-        self.state2policy = self.state2policy_mlp
-        self.statetorch2policy = self.statetorch2policy_mlp
-        # self.statetorch2proxy = self.statetorch2policy
+        if policy_type == "mlp":
+            self.state2policy = self.state2policy_mlp
+            self.statetorch2policy = self.statetorch2policy_mlp
+        elif policy_type != "gnn":
+            raise ValueError(
+                f"Unrecognized policy_type = {policy_type}, expected either 'mlp' or 'gnn'."
+            )
```

Comment:
What happens if `policy_type` is `gnn`, is it in the other PR?

Reply:
Then we use default `state2policy` (returning complete state), which is converted to a graph in the policy itself (to avoid issues with current GFN code expecting tensors to be passed around).

Reply:
OK I see, I forgot the conversion happens _in_ the policy.